client/signup/steps/website-content/index.tsx

```diff
@@ -300,6 +302,13 @@ export default function WrapperWebsiteContent(
 		}
 	}, [ data, queryObject.siteSlug ] );
 
+	useEffect( () => {
+		if ( skippedCheckout === '1' ) {
```

Comment:
`Nonblocking:` 

This seems totally reasonable for a patch.

I'm a bit surprised this isn't already handled at a level above individual steps 🤔

Reply:
Yeah it's not necessarily a bad thing to expect the checkout destination to handle both the the success and cancelation cases. You could imagine the flow doing something nice like acknowledging they didn't pay, offering a discount, giving them alternative solutions etc.
On the other hand, it could save a redirect bounce if you could set the `cancel_to` param up front 🤷‍♂️